README.md

```diff
@@ -58,5 +77,14 @@ console.log(evens) // [10, 4, 6, 8, 2]
 
 function reverse(array) {
   // -- YOUR ANSWER HERE --
+   let length = array.length
+  let bucket = [];
+  
+
+  for(var i=0; i<length/2; i++) {
+    let first = i;
+    let second = array.length-1 -[i];
+    swap(array, first, second);  // calling function
+  }
```

Comment:
Perfect.  Except that you can get rid of the `bucket` variable since you never use it.